dist/api_data.py

```diff
@@ -1699,6 +1699,10 @@ def __ge__(self, other):
         Display the contents of in-memory pages as they are verified,
         using the application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_app_data', 'false', r'''
+        Display application data as pages or blocks are verified,
+        using the application's message handler, intended for debugging.''',
+        type='boolean'),
```

Comment:
Could we default this to on? I'd rather not change the default behavior. Or was there an explicit request to disable by default?